service-tests/npm.js

```diff
@@ -106,9 +158,7 @@ t.create('license when registry returns an invalid JSON')
   .get('/l/invalid-json.json?style=_shields_test')
   .intercept(nock => nock('https://registry.npmjs.org')
     .get('/invalid-json/latest')
-    .reply(200, 'invalid json'), {
-      'Content-Type': 'application/json'
-    })
+    .reply(invalidJSON))
```

Comment:
Yay for refactoring. This looks great!

Would you be willing to go through the other service tests and update them to use your new fixture as well?